modules/custom/az_migration/src/Plugin/migrate/process/ParagraphsBehavior.php

```diff
@@ -62,6 +62,45 @@ public function transform($value, MigrateExecutableInterface $migrate_executable
       $behavior['az_display_settings']['card_width_xs'] = $this->configuration['card_width_xs'];
     }
 
+    // Getting the card width checking field.
+    if (!empty($this->configuration['card_width_field'])) {
+      $card_width_field = $row->getSourceProperty($this->configuration['card_width_field']);
+      $card_count_field = $row->getSourceProperty($this->configuration['card_count_field']);
+
+      // If the card width field value is Full Width, adding the conditions.
+      if (!empty($card_width_field[0]['value']) &&
+        $card_width_field[0]['value'] === 'full_width') {
+
+        $card_width = [
+          '1' => 'col-md-12 col-lg-12',
+          '2' => 'col-md-6 col-lg-6',
+          '3' => 'col-md-4 col-lg-4',
+          '4' => 'col-md-3 col-lg-3',
+        ];
+
+        $card_width_sm = [
+          '1' => 'col-sm-12',
+          '2' => 'col-sm-6',
+          '3' => 'col-sm-4',
```

Comment:
If 3 cards have content, cards per row on tablet should be set to 2, not 3.
```suggestion
          '3' => 'col-sm-6',
```